src/components/OverviewCard.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import elementStyles from '../styles/isomer-cms/Elements.module.scss';
+import contentStyles from '../styles/isomer-cms/pages/Content.module.scss';
+
+const OverviewCard = ({
+  title, date, category, settingsToggle, itemIndex, siteName, collectionName, fileName, isHomepage = false
+}) => {
+  const link = isHomepage ? `/sites/${siteName}/homepage` :
+    collectionName
+    ? `/sites/${siteName}/collections/${collectionName}/${fileName}`
+    : `/sites/${siteName}/pages/${fileName}`
```

Comment:
Missing logic for resources page. Currently, if you click into this from resources, you attempt to retrieve an unknown page from the pages endpoint

Reply:
We can use a small function in the same file instead of an overly complicated ternary which is hard to read

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/184/commits/e36e346e1ad587c3a3ca7e2a97f464165de637ca